saml/build.gradle

```diff
@@ -1,12 +1,12 @@
 dependencies {
-    compile 'com.auth0:java-jwt'
-    compile 'net.shibboleth.utilities:java-support'
-    compile 'org.opensaml:opensaml-core'
-    compile 'org.opensaml:opensaml-saml-api'
-    compile 'org.opensaml:opensaml-saml-impl'
-    compile 'org.opensaml:opensaml-messaging-api'
-    compile 'org.opensaml:opensaml-messaging-impl'
-    compile 'org.opensaml:opensaml-soap-api'
-    compile 'org.opensaml:opensaml-soap-impl'
-    testCompile 'org.jsoup:jsoup'
+    api 'com.auth0:java-jwt'
```

Comment:
I was somewhat surprised to find this in our public API, but looking at it realized it is tedious if we wanted to separate. Something to think about.

Also, I use `java-jwt` too and it's probably a fine choice, but worth giving a look at other libraries too I guess, maybe https://github.com/google/tink/blob/master/docs/JAVA-HOWTO.md